target/linux/ar71xx/patches-4.9/940-MIPS-Add-support-for-ARCH_MMAP_RND_-COMPAT_-BITS.patch

```diff
@@ -0,0 +1,96 @@
+Backport from linux-4.11
+
```

Comment:
I think these patches should go to generic so all mips targets can benefit from this feature.